cmd/node_builder.go

```diff
@@ -283,6 +284,7 @@ func DefaultBaseConfig() *BaseConfig {
 		secretsdir:       NotSet,
 		secretsDBEnabled: true,
 		level:            "info",
+		debugLogLimit:    1000,
```

Comment:
I'm not sure what an appropriate default is.

Reply:
maybe this is a bit low. I just checked access-001  is producing 500 per second right now

Reply:
looking over the logs from the LN outage, it looks like nodes were peaking around 500 events/sec. It's hard to say. I updated the limit to 2000, but operators will likely need to tune this for their hardware